CMakeLists.txt

```
@@ -8,6 +8,13 @@ set (alldirs
   GEOSwgcm_GridComp
   )
 
+option(BUILD_WITH_GIGATRAJ "Build GEOSgcm with Gigatraj" OFF)
+
+if (BUILD_WITH_GIGATRAJ)
+  add_definitions (-DHAS_GIGATRAJ)
+  set (alldirs ${alldirs} GEOSgigatraj_GridComp) 
```

This is better here:
```suggestion
  list(APPEND alldirs GEOSgigatraj_GridComp) 
```